Description: fix quite a few typos in the code and user-oriented text
Author: Julien Puydt
Forwarded: yes

--- giac.orig/src/aspen_translate.h
+++ giac/src/aspen_translate.h
@@ -290,7 +290,7 @@
 {"For must have 4 args",0,"For 循环必须具有 4 个宗量","for doit avoir 4 args","For muss 4 Argumente haben","Para debe tener 4 args","For moet 4 argumenten hebben","Deve ter 4 argumentos","Forには引数が4つ必要です"},
 {"For: Unable to check test",0,"For  循环: 无法检查检验","For: impossible de faire le test","For: Testprüfung nicht möglich","Para: No es posible verificar la prueba","For: kan test niet controleren","For: não é possível verificar o teste","For: 検定を確認できません"},
 {"Function not defined",0,"未定义函数","Fonction non définie","Funktion nicht definiert","Función no definida","Functie niet gedefinieerd","Função não definida","関数は定義されていません"},
-{"GCD not successfull",0,"最大公约数失败","PGCD: échec","GGT nicht erfolgreich","GCD sin éxito","GCD is mislukt","GCD sem êxito","GCDは成功しませんでした"},
+{"GCD not successful",0,"最大公约数失败","PGCD: échec","GGT nicht erfolgreich","GCD sin éxito","GCD is mislukt","GCD sem êxito","GCDは成功しませんでした"},
 {"GF inv internal bug",0,"伽罗瓦域求矩阵逆内部错误","GF inv bug interne","GF-Inv. - interner Fehler","Error interno de inv de GF","Interne fout GF inv","Erro interno GF inv","GF inv内部バグ"},
 {"GF is used for non-prime finite field. Use % or mod for prime fields, e.g. 1 % ",0,"GF 用于非本原有限域。将 % 或 mod 用于素域，例如 1 % ","GF est utilisé pour les corps finis non premier. Utiliser % ou mod pour les corps premiers, par ex 1","GF wird für Nicht-Primzahlfelder verwendet. % oder mod für Primzahlfelder verwenden, z. B. 1 % ","GF se usa en el campo finito no primo. Use % o el mod para los campos primos, por ej. 1 % ","GF wordt gebruikt voor een eindig veld zonder priemgetal. Gebruik % of mod voor priemgetalvelden, bijv. 1 % ","GF é utilizado para um campo finito não primo. Utilize % ou mod para os campos primos, por exemplo, 1 % ","GFは非素数有限フィールドに使用されます。素数フィールドには%またはmodを使用してください（例: 1 %） "},
 {"GF polyfactor",0,"伽罗瓦域多项式","GF polyfactor","GF-Polyfaktor","Polifactor de GF","GF polyfactor","GF polyfactor","GF多係数"},
@@ -520,7 +520,7 @@
 {"Psi/Zeta/Eta: bad second argument",0,"Psi/Zeta/Eta: 第二个宗量错误","Psi/Zeta/Eta: second argument incorrect","Psi/Zeta/Eta: ungültiges zweites Argument","Psi/Zeta/Eta: segundo argumento incorrecto","Psi/Zeta/Eta: ongeldig tweede argument","Psi/Zeta/Eta: segundo argumento errado","Psi/Zeta/Eta: 不正な第2引数"},
 {"Quadratic sieve failure, perhaps number too large",0,"二次筛法失败，可能数字过大","Quadratic sieve failure, perhaps number too large","Fehler bei quadr. Siebung, Anzahl u. U. zu groß","Falla de criba cuadrática, tal vez el número es demasiado grande","Fout kwadratische zeef, getal is mogelijk te groot","Falha de Sieve quadrático, talvez número demasiado grande","二次のふるいは失敗しました、おそらく数が大きすぎます"},
 {"Quaternion has 1 or 4 arguments",0,"四元数有 1 或 4 个宗量","Quaternion a 1 ou 4 arguments","Quaternion hat 1 oder 4 Argumente","El cuaternio tiene 1 o 4 argumentos","Quaternion heeft 1 of 4 argumenten","O quaternião tem 1 ou 4 argumentos","四元数は1つまたは4つの引数を持っています"},
-{"RK8 evolve not successfull",0,"RK8 推演失败","Echec de résolution par RK8","RK8-Entwicklung nicht erfolgreich","La evolución de RK8 no fue exitosa","RK8 evolve mislukt","Evolução RK8 sem êxito","RK8展開は成功しませんでした"},
+{"RK8 evolve not successful",0,"RK8 推演失败","Echec de résolution par RK8","RK8-Entwicklung nicht erfolgreich","La evolución de RK8 no fue exitosa","RK8 evolve mislukt","Evolução RK8 sem êxito","RK8展開は成功しませんでした"},
 {"Re",0,"Re","re","Re","Re","Re","Re","Re"},
 {"Reconstructed denominator is not prime with modulo",0,"重构的分母不是具有模数的素数","Le dénominateur reconstruit est non premier avec modulo","Rekonstruierter Nenner ist nicht teilerfremd mit modulo","El denominador reconstruido no es primo con módulo","Gereconstrueerde teller is geen priemgetal met modulo","Denominador reconstruído não é primo com módulo","再構築された分母は法と素ではありません"},
 {"Reserved word:",0,"保留字:","Mot-clef réservé","Reserviertes Wort:","Palabra reservada:","Gereserveerd woord:","Palavra reservada:","予約語:"},
@@ -530,7 +530,7 @@
 {"Risch diff dimension",0,"Risch 微分维数",0,"Risch-Differentiation - Dimension","Dimensión dif de Risch","Risch-differentiaaldimensie","Dimensão diferencial de Risch","リッシュ差次元"},
 {"Run convert(matrix,array) for dense conversion",0,"运行 convert(matrix,array) 以进行密集转换","Tapez convert(matrix,array) pour avoir une matrice dense","convert(Matrix,Array) für dichte Konvertierung ausführen","Ejecute convert(matrix,array) para la conversión densa","Voer convert(matrix,array) uit voor dichte conversie","Executar convert(matrix,array) para conversão densa","稠密な変換のためにconvert(matrix,array)を実行してください"},
 {"Running TI89 text translation __.ti",0,"运行 TI89 文本翻译 __.ti","Exécute la traduction TI89, fichier __.ti","Ausführung von TI89-Textübersetzung __.map","Ejecución de translation __.ti del texto TI89","TI89 tekstomzetting __.ti wordt uitgevoerd","Executar translação de texto TI89 __.ti","TI89テキスト変換__.tiの実行"},
-{"Running a probabilistic check for the reconstructed Groebner basis. If successfull, error probability is less than ",0,"为重构的 Groebner 基运行概率检查。如果成功，错误概率将小于 ","Exécution d'une vérification non déterministe pour la base de Groebner reconstruire. Si le test réussit, la probabilité d'avoir une réponse fausse est inférieure à ","Ausführung einer probabilistischen Prüfung für die neu erstellte Gröbnerbasis. Wenn erfolgreich, ist die Fehlerwahrscheinlichkeit kleiner als ","Se está ejecutando una verificación de probabilidades para la base de Groebner reconstruida. Si tiene éxito, la probabilidad de error es inferior a ","Er wordt een probabilistische controle uitgevoerd voor de gereconstrueerde Gröbner-basis. Indien succesvol, is de foutkans kleiner dan ","Executar uma verificação probabilística para a base de Groebner reconstruída. Se for bem-sucedida, a probabilidade de erro é inferior a ","再構築されたグレブナー基底について確率論的な確認を実行します。成功した場合、エラー確率は未満です "},
+{"Running a probabilistic check for the reconstructed Groebner basis. If successful, error probability is less than ",0,"为重构的 Groebner 基运行概率检查。如果成功，错误概率将小于 ","Exécution d'une vérification non déterministe pour la base de Groebner reconstruire. Si le test réussit, la probabilité d'avoir une réponse fausse est inférieure à ","Ausführung einer probabilistischen Prüfung für die neu erstellte Gröbnerbasis. Wenn erfolgreich, ist die Fehlerwahrscheinlichkeit kleiner als ","Se está ejecutando una verificación de probabilidades para la base de Groebner reconstruida. Si tiene éxito, la probabilidad de error es inferior a ","Er wordt een probabilistische controle uitgevoerd voor de gereconstrueerde Gröbner-basis. Indien succesvol, is de foutkans kleiner dan ","Executar uma verificação probabilística para a base de Groebner reconstruída. Se for bem-sucedida, a probabilidade de erro é inferior a ","再構築されたグレブナー基底について確率論的な確認を実行します。成功した場合、エラー確率は未満です "},
 {"Running in secure mode",0,"在安全模式下运行","Exécution en mode sécure","Ausführung im geschützten Modus","Ejecución en modo seguro","Veilige modus actief","Executar em modo seguro","セキュア モードで実行"},
 {"Running maple text translation __.map",0,"运行 maple 文本翻译 __.map","Exécute la traduction Maple, fichier __.map","Ausführung von Maple-Textübersetzung __.map","Ejecución de translation __.map del texto maple","Maple tekstomzetting __.map wordt uitgevoerd","Executar translação de texto maple __.map","mapleテキスト変換__.mapの実行"},
 {"Running non recursive evaluator",0,"运行非递归求值程序","Exécution en mode d'évaluation non récursive","Ausführung nicht-rekursive Auswertung","Ejecutando evaluador no recursivo","Niet-recursieve evaluator wordt uitgevoerd","Executar avaliador não recursivo","非再帰的エバリュエーターの実行"},
@@ -662,8 +662,8 @@
 {"Underflow",0,"下溢","Nombre trop petit","Unterlauf","Desbordamiento de capacidad mínima","Onderloop","Abaixo do limite","アンダーフロー"},
 {"Uniform density, estimating interval boundaries from data ",0,"均匀密度，根据数据估计区间边界 ","Densité uniforme, estimation de l'intervalle par les données","Einheitliche Dichte, Schätzung der Intervallgrenzen aus Daten ","Densidad uniforme, calculando los límites del intervalo a partir de los datos ","Uniforme dichtheid, schatting van intervalgrenzen op basis van data ","Densidade uniforme, estimando os limites do intervalo a partir de dados ","一様な密度、データから区間境界を推定 "},
 {"Union",0,"联合","union","Vereinigung","Unión","Union","União","結合"},
-{"Unitialized local variable ",0,"未初始化的局部变量 ","Variable locale non initialisée ","Nicht initialisierte lokale Variable ","Variable local no inicializada ","Niet-geinitialiseerde lokale variabele ","Variável local não inicializada ","Unitializedローカル変数 "},
-{"Unitialized parameter ",0,"未初始化的参数 ","Paramètre non initialisé ","Nicht initialisierter Parameter ","Parámetro no inicializado ","Niet-geïnitialiseerde parameter ","Parâmetro não inicializado ","Unitializedパラメーター "},
+{"Uninitialized local variable ",0,"未初始化的局部变量 ","Variable locale non initialisée ","Nicht initialisierte lokale Variable ","Variable local no inicializada ","Niet-geinitialiseerde lokale variabele ","Variável local não inicializada ","Uninitializedローカル変数 "},
+{"Uninitialized parameter ",0,"未初始化的参数 ","Paramètre non initialisé ","Nicht initialisierter Parameter ","Parámetro no inicializado ","Niet-geïnitialiseerde parameter ","Parâmetro não inicializado ","Uninitializedパラメーター "},
 {"Unknown instruction tag:",0,"未知说明标签:","Tag inconnu:","Unbekanntes Anweisungstag:","Etiqueta de instrucción desconocida:","Onbekende instructietag:","Etiqueta de instruções desconhecida:","不明な指示タグ:"},
 {"Unknown secondary tag",0,"未知副标签","Tag secondaire inconnu","Unbekanntes sekundäres Tag","Etiqueta secundaria desconocida","Onbekende secundaire tag","Etiqueta secundária desconhecida","不明な第2タグ"},
 {"Unknown subsop rule ",0,"未知 subsop 规则 ","subsop : règle inconnue ","Unbekannte subsop-Regel ","Regla de subsop desconocida ","Onbekende subsop regel ","Regra subsop desconhecida ","不明なsubsop規則 "},
--- giac.orig/src/help.cc
+++ giac/src/help.cc
@@ -1077,7 +1077,7 @@
 	  break;
 	}
 	if (s>8 && tmp.substr(s-8,8)=="</B></A>"){
-	  // Find backward the first occurence of <A
+	  // Find backward the first occurrence of <A
 	  int l=s-8;
 	  for (;l>0;--l){
 	    if (tmp[l]=='<' && tmp[l+1]=='A')
--- giac.orig/src/intg.cc
+++ giac/src/intg.cc
@@ -5841,7 +5841,7 @@
 					     &t, t1, &h,
 					     y);
 	if (status != GSL_SUCCESS)
-	  return gensizeerr(gettext("RK8 evolve not successfull"));
+	  return gensizeerr(gettext("RK8 evolve not successful"));
 	if (debug_infolevel>5)
 	  CERR << nstep << ":" << t << ",y5=" << double2vecteur(y,dim) << '\n';
 	if (return_curve)  {
--- giac.orig/src/kdisplay.cc
+++ giac/src/kdisplay.cc
@@ -1188,12 +1188,12 @@
   {"avance n", "avance ", "Turtle forward n steps, default n=10", "#avance 30", 0, CAT_CATEGORY_LOGO},
   {"axes", "axes", "Axes visible or not axes=1 or 0", "#axes=0", 0, CAT_CATEGORY_PROGCMD << 8|XCAS_ONLY},
   {"baisse_crayon ", "baisse_crayon ", "Turtle moves with the pen writing.", 0, 0, CAT_CATEGORY_LOGO},
-  {"barplot(list)", 0, "Bar plot of 1-d statistic serie data in list.", "[3/2,2,1,1/2,3,2,3/2]", 0, CAT_CATEGORY_STATS},
+  {"barplot(list)", 0, "Bar plot of 1-d statistic series data in list.", "[3/2,2,1,1/2,3,2,3/2]", 0, CAT_CATEGORY_STATS},
   {"binomial(n,p,k)", 0, "binomial(n,p,k) probability to get k success with n trials where p is the probability of success of 1 trial. binomial_cdf(n,p,k) is the probability to get at most k successes. binomial_icdf(n,p,t) returns the smallest k such that binomial_cdf(n,p,k)>=t", "10,.5,4", 0, CAT_CATEGORY_PROBA},
   {"bitxor", "bitxor", "Exclusive or", "#bitxor(1,2)", 0, CAT_CATEGORY_PROGCMD},
   {"black", "black", "Display option", "#display=black", 0, CAT_CATEGORY_PROGCMD},
   {"blue", "blue", "Display option", "#display=blue", 0, CAT_CATEGORY_PROGCMD},
-  {"camembert(list)", 0, "Camembert pie-chart of a 1-d statistical serie.", "[[\"France\",6],[\"Germany\",12],[\"Switzerland\",5]]", 0, CAT_CATEGORY_STATS},
+  {"camembert(list)", 0, "Camembert pie-chart of a 1-d statistical series.", "[[\"France\",6],[\"Germany\",12],[\"Switzerland\",5]]", 0, CAT_CATEGORY_STATS},
   {"cache_tortue ", "cache_tortue ", "Hide turtle (once the picture has been drawn).", 0, 0, CAT_CATEGORY_LOGO},
   {"ceil(x)", 0, "Smallest integer not less than x", "1.2", 0, CAT_CATEGORY_REAL},
   {"cfactor(p)", 0, "Factorization over C.", "x^4-1", 0, CAT_CATEGORY_ALGEBRA | (CAT_CATEGORY_COMPLEXNUM << 8)},
@@ -1262,7 +1262,7 @@
   {"gcd(a,b,...)", 0, "Greatest common divisor. See also iegcd and egcd for extended GCD.", "23,13", "x^2-1,x^3-1", CAT_CATEGORY_ARIT | (CAT_CATEGORY_POLYNOMIAL << 8)},
   {"gl_x", "gl_x", "Display settings X gl_x=xmin..xmax", "#gl_x=0..2", 0, CAT_CATEGORY_PROGCMD},
   {"gl_y", "gl_y", "Display settings Y gl_y=ymin..ymax", "#gl_y=-1..1", 0, CAT_CATEGORY_PROGCMD},
-  {"gramschmidt(M)", 0, "Gram-Schmidt orthonormalization (line vectors or linearly independant set of vectors)", "[[1,2,3],[4,5,6]]", "[1,1+x],(p,q)->integrate(p*q,x,-1,1)", CAT_CATEGORY_LINALG},
+  {"gramschmidt(M)", 0, "Gram-Schmidt orthonormalization (line vectors or linearly independent set of vectors)", "[[1,2,3],[4,5,6]]", "[1,1+x],(p,q)->integrate(p*q,x,-1,1)", CAT_CATEGORY_LINALG},
   {"green", "green", "Display option", "#display=green", 0, CAT_CATEGORY_PROGCMD},
   {"halftan(expr)", 0, "Convert cos, sin, tan with tan(angle/2).","cos(x)", 0, CAT_CATEGORY_TRIG},
   {"hermite(n)", 0, "n-th Hermite polynomial", "10", 0, CAT_CATEGORY_POLYNOMIAL},
@@ -1324,7 +1324,7 @@
   {"plotarea(expr,x=a..b,[n,meth])", 0, "Area under curve with specified quadrature.", "1/x,x=1..3,2,trapezoid", 0, CAT_CATEGORY_PLOT},
 #endif
   {"plotcontour(expr,[x=xm..xM,y=ym..yM],levels)", 0, "Levels of expr.", "x^2+2y^2,[x=-2..2,y=-2..2],[1,2]", 0, CAT_CATEGORY_PLOT},
-  {"plotfield(f(t,y),[t=tmin..tmax,y=ymin..ymax])", 0, "Plot field of differential equation y'=f(t,y), an optionnally one solution by adding plotode=[t0,y0]", "sin(t*y),[t=-3..3,y=-3..3],plotode=[0,1]", 0, CAT_CATEGORY_PLOT},
+  {"plotfield(f(t,y),[t=tmin..tmax,y=ymin..ymax])", 0, "Plot field of differential equation y'=f(t,y), an optionally one solution by adding plotode=[t0,y0]", "sin(t*y),[t=-3..3,y=-3..3],plotode=[0,1]", 0, CAT_CATEGORY_PLOT},
   {"plotlist(list)", 0, "Plot a list", "[3/2,2,1,1/2,3,2,3/2]", 0, CAT_CATEGORY_PLOT},
   {"plotode(f(t,y),[t=tmin..tmax,y],[t0,y0])", 0, "Plot solution of differential equation y'=f(t,y), y(t0)=y0.", "sin(t*y),[t=-3..3,y],[0,1]", 0, CAT_CATEGORY_PLOT},
   {"plotparam([x,y],t)", 0, "Parametric plot. For example plotparam([sin(3t),cos(2t)],t,0,pi) or plotparam(exp(i*t),t,0,pi)", "[sin(3t),cos(2t)],t,0,pi", "[t^2,t^3],t=-1..1,tstep=0.1", CAT_CATEGORY_PLOT},
@@ -3668,7 +3668,7 @@
 
   // vertical merge with same baseline
   // for vertical merge of hp,yp at top (like ^) add fontsize to yp
-  // at bottom (like lower bound of int) substract fontsize from yp
+  // at bottom (like lower bound of int) subtract fontsize from yp
   void Equation_vertical_adjust(int hp,int yp,int & h,int & y){
     int yf=min(y,yp);
     h=max(y+h,yp+hp)-yf;
@@ -5141,7 +5141,7 @@
 	      fl_pie(deltax+round(i0-i1),deltay+round(j0-j1),round(2*i1),round(2*j1),full?0:anglei*180/M_PI+.5,full?360:anglef*180/M_PI+.5,couleur,false);
 	    else {
 	      fl_arc(deltax+round(i0-i1),deltay+round(j0-j1),round(2*i1),round(2*j1),full?0:anglei*180/M_PI+.5,full?360:anglef*180/M_PI+.5,couleur);
-	      if (v.size()>=4){ // if cercle has the optionnal 5th arg
+	      if (v.size()>=4){ // if cercle has the optional 5th arg
 		if (v[3]==2)
 		  petite_fleche(i0+i1*std::cos(anglem),j0-j1*std::sin(anglem),-i1*std::sin(anglem),-j1*std::cos(anglem),deltax,deltay,width,couleur);
 		else {
--- giac.orig/src/numworks_translate.h
+++ giac/src/numworks_translate.h
@@ -290,7 +290,7 @@
 {"For must have 4 args",0,"For 循环必须具有 4 个宗量","for doit avoir 4 args","For muss 4 Argumente haben","Para debe tener 4 args","For moet 4 argumenten hebben","Deve ter 4 argumentos","Forには引数が4つ必要です"},
 {"For: Unable to check test",0,"For  循环: 无法检查检验","For: impossible de faire le test","For: Testprüfung nicht möglich","Para: No es posible verificar la prueba","For: kan test niet controleren","For: não e possível verificar o teste","For: 検定を確認できません"},
 {"Function not defined",0,"未定义函数","Fonction non definie","Funktion nicht definiert","Función no definida","Functie niet gedefinieerd","Funcão não definida","関数は定義されていません"},
-{"GCD not successfull",0,"最大公约数失败","PGCD: echec","GGT nicht erfolgreich","GCD sin exito","GCD is mislukt","GCD sem exito","GCDは成功しませんでした"},
+{"GCD not successful",0,"最大公约数失败","PGCD: echec","GGT nicht erfolgreich","GCD sin exito","GCD is mislukt","GCD sem exito","GCDは成功しませんでした"},
 {"GF inv internal bug",0,"伽罗瓦域求矩阵逆内部错误","GF inv bug interne","GF-Inv. - interner Fehler","Error interno de inv de GF","Interne fout GF inv","Erro interno GF inv","GF inv内部バグ"},
 {"GF is used for non-prime finite field. Use % or mod for prime fields, e.g. 1 % ",0,"GF 用于非本原有限域。将 % 或 mod 用于素域，例如 1 % ","GF est utilise pour les corps finis non premier. Utiliser % ou mod pour les corps premiers, par ex 1","GF wird für Nicht-Primzahlfelder verwendet. % oder mod für Primzahlfelder verwenden, z. B. 1 % ","GF se usa en el campo finito no primo. Use % o el mod para los campos primos, por ej. 1 % ","GF wordt gebruikt voor een eindig veld zonder priemgetal. Gebruik % of mod voor priemgetalvelden, bijv. 1 % ","GF e utilizado para um campo finito não primo. Utilize % ou mod para os campos primos, por exemplo, 1 % ","GFは非素数有限フィールドに使用されます。素数フィールドには%またはmodを使用してください（例: 1 %） "},
 {"GF polyfactor",0,"伽罗瓦域多项式","GF polyfactor","GF-Polyfaktor","Polifactor de GF","GF polyfactor","GF polyfactor","GF多係数"},
@@ -520,7 +520,7 @@
 {"Psi/Zeta/Eta: bad second argument",0,"Psi/Zeta/Eta: 第二个宗量错误","Psi/Zeta/Eta: second argument incorrect","Psi/Zeta/Eta: ungültiges zweites Argument","Psi/Zeta/Eta: segundo argumento incorrecto","Psi/Zeta/Eta: ongeldig tweede argument","Psi/Zeta/Eta: segundo argumento errado","Psi/Zeta/Eta: 不正な第2引数"},
 {"Quadratic sieve failure, perhaps number too large",0,"二次筛法失败，可能数字过大","Quadratic sieve failure, perhaps number too large","Fehler bei quadr. Siebung, Anzahl u. U. zu groß","Falla de criba cuadrática, tal vez el número es demasiado grande","Fout kwadratische zeef, getal is mogelijk te groot","Falha de Sieve quadrático, talvez número demasiado grande","二次のふるいは失敗しました、おそらく数が大きすぎます"},
 {"Quaternion has 1 or 4 arguments",0,"四元数有 1 或 4 个宗量","Quaternion a 1 ou 4 arguments","Quaternion hat 1 oder 4 Argumente","El cuaternio tiene 1 o 4 argumentos","Quaternion heeft 1 of 4 argumenten","O quaternião tem 1 ou 4 argumentos","四元数は1つまたは4つの引数を持っています"},
-{"RK8 evolve not successfull",0,"RK8 推演失败","Echec de resolution par RK8","RK8-Entwicklung nicht erfolgreich","La evolución de RK8 no fue exitosa","RK8 evolve mislukt","Evolucão RK8 sem exito","RK8展開は成功しませんでした"},
+{"RK8 evolve not successful",0,"RK8 推演失败","Echec de resolution par RK8","RK8-Entwicklung nicht erfolgreich","La evolución de RK8 no fue exitosa","RK8 evolve mislukt","Evolucão RK8 sem exito","RK8展開は成功しませんでした"},
 {"Re",0,"Re","re","Re","Re","Re","Re","Re"},
 {"Reconstructed denominator is not prime with modulo",0,"重构的分母不是具有模数的素数","Le denominateur reconstruit est non premier avec modulo","Rekonstruierter Nenner ist nicht teilerfremd mit modulo","El denominador reconstruido no es primo con módulo","Gereconstrueerde teller is geen priemgetal met modulo","Denominador reconstruído não e primo com módulo","再構築された分母は法と素ではありません"},
 {"Reserved word:",0,"保留字:","Mot-clef reserve","Reserviertes Wort:","Palabra reservada:","Gereserveerd woord:","Palavra reservada:","予約語:"},
@@ -530,7 +530,7 @@
 {"Risch diff dimension",0,"Risch 微分维数",0,"Risch-Differentiation - Dimension","Dimensión dif de Risch","Risch-differentiaaldimensie","Dimensão diferencial de Risch","リッシュ差次元"},
 {"Run convert(matrix,array) for dense conversion",0,"运行 convert(matrix,array) 以进行密集转换","Tapez convert(matrix,array) pour avoir une matrice dense","convert(Matrix,Array) für dichte Konvertierung ausführen","Ejecute convert(matrix,array) para la conversión densa","Voer convert(matrix,array) uit voor dichte conversie","Executar convert(matrix,array) para conversão densa","稠密な変換のためにconvert(matrix,array)を実行してください"},
 {"Running TI89 text translation __.ti",0,"运行 TI89 文本翻译 __.ti","Execute la traduction TI89, fichier __.ti","Ausführung von TI89-Textübersetzung __.map","Ejecución de translation __.ti del texto TI89","TI89 tekstomzetting __.ti wordt uitgevoerd","Executar translacão de texto TI89 __.ti","TI89テキスト変換__.tiの実行"},
-{"Running a probabilistic check for the reconstructed Groebner basis. If successfull, error probability is less than ",0,"为重构的 Groebner 基运行概率检查。如果成功，错误概率将小于 ","Execution d'une verification non deterministe pour la base de Groebner reconstruire. Si le test reussit, la probabilite d'avoir une reponse fausse est inferieure a ","Ausführung einer probabilistischen Prüfung für die neu erstellte Gröbnerbasis. Wenn erfolgreich, ist die Fehlerwahrscheinlichkeit kleiner als ","Se está ejecutando una verificación de probabilidades para la base de Groebner reconstruida. Si tiene exito, la probabilidad de error es inferior a ","Er wordt een probabilistische controle uitgevoerd voor de gereconstrueerde Gröbner-basis. Indien succesvol, is de foutkans kleiner dan ","Executar uma verificacão probabilística para a base de Groebner reconstruída. Se for bem-sucedida, a probabilidade de erro e inferior a ","再構築されたグレブナー基底について確率論的な確認を実行します。成功した場合、エラー確率は未満です "},
+{"Running a probabilistic check for the reconstructed Groebner basis. If successful, error probability is less than ",0,"为重构的 Groebner 基运行概率检查。如果成功，错误概率将小于 ","Execution d'une verification non deterministe pour la base de Groebner reconstruire. Si le test reussit, la probabilite d'avoir une reponse fausse est inferieure a ","Ausführung einer probabilistischen Prüfung für die neu erstellte Gröbnerbasis. Wenn erfolgreich, ist die Fehlerwahrscheinlichkeit kleiner als ","Se está ejecutando una verificación de probabilidades para la base de Groebner reconstruida. Si tiene exito, la probabilidad de error es inferior a ","Er wordt een probabilistische controle uitgevoerd voor de gereconstrueerde Gröbner-basis. Indien succesvol, is de foutkans kleiner dan ","Executar uma verificacão probabilística para a base de Groebner reconstruída. Se for bem-sucedida, a probabilidade de erro e inferior a ","再構築されたグレブナー基底について確率論的な確認を実行します。成功した場合、エラー確率は未満です "},
 {"Running in secure mode",0,"在安全模式下运行","Execution en mode secure","Ausführung im geschützten Modus","Ejecución en modo seguro","Veilige modus actief","Executar em modo seguro","セキュア モードで実行"},
 {"Running maple text translation __.map",0,"运行 maple 文本翻译 __.map","Execute la traduction Maple, fichier __.map","Ausführung von Maple-Textübersetzung __.map","Ejecución de translation __.map del texto maple","Maple tekstomzetting __.map wordt uitgevoerd","Executar translacão de texto maple __.map","mapleテキスト変換__.mapの実行"},
 {"Running non recursive evaluator",0,"运行非递归求值程序","Execution en mode d'evaluation non recursive","Ausführung nicht-rekursive Auswertung","Ejecutando evaluador no recursivo","Niet-recursieve evaluator wordt uitgevoerd","Executar avaliador não recursivo","非再帰的エバリュエーターの実行"},
@@ -662,8 +662,8 @@
 {"Underflow",0,"下溢","Nombre trop petit","Unterlauf","Desbordamiento de capacidad mínima","Onderloop","Abaixo do limite","アンダーフロー"},
 {"Uniform density, estimating interval boundaries from data ",0,"均匀密度，根据数据估计区间边界 ","Densite uniforme, estimation de l'intervalle par les donnees","Einheitliche Dichte, Schätzung der Intervallgrenzen aus Daten ","Densidad uniforme, calculando los límites del intervalo a partir de los datos ","Uniforme dichtheid, schatting van intervalgrenzen op basis van data ","Densidade uniforme, estimando os limites do intervalo a partir de dados ","一様な密度、データから区間境界を推定 "},
 {"Union",0,"联合","union","Vereinigung","Unión","Union","União","結合"},
-{"Unitialized local variable ",0,"未初始化的局部变量 ","Variable locale non initialisee ","Nicht initialisierte lokale Variable ","Variable local no inicializada ","Niet-geinitialiseerde lokale variabele ","Variável local não inicializada ","Unitializedローカル変数 "},
-{"Unitialized parameter ",0,"未初始化的参数 ","Parametre non initialise ","Nicht initialisierter Parameter ","Parámetro no inicializado ","Niet-geïnitialiseerde parameter ","Parâmetro não inicializado ","Unitializedパラメーター "},
+{"Uninitialized local variable ",0,"未初始化的局部变量 ","Variable locale non initialisee ","Nicht initialisierte lokale Variable ","Variable local no inicializada ","Niet-geinitialiseerde lokale variabele ","Variável local não inicializada ","Uninitializedローカル変数 "},
+{"Uninitialized parameter ",0,"未初始化的参数 ","Parametre non initialise ","Nicht initialisierter Parameter ","Parámetro no inicializado ","Niet-geïnitialiseerde parameter ","Parâmetro não inicializado ","Uninitializedパラメーター "},
 {"Unknown instruction tag:",0,"未知说明标签:","Tag inconnu:","Unbekanntes Anweisungstag:","Etiqueta de instrucción desconocida:","Onbekende instructietag:","Etiqueta de instrucões desconhecida:","不明な指示タグ:"},
 {"Unknown secondary tag",0,"未知副标签","Tag secondaire inconnu","Unbekanntes sekundäres Tag","Etiqueta secundaria desconocida","Onbekende secundaire tag","Etiqueta secundária desconhecida","不明な第2タグ"},
 {"Unknown subsop rule ",0,"未知 subsop 规则 ","subsop : regle inconnue ","Unbekannte subsop-Regel ","Regla de subsop desconocida ","Onbekende subsop regel ","Regra subsop desconhecida ","不明なsubsop規則 "},
--- giac.orig/src/solve.cc
+++ giac/src/solve.cc
@@ -7553,7 +7553,7 @@
 	  }
 	}
 	if (1 || curdeg==1){
-	  // Choose lowest number of dependant equations in poselim
+	  // Choose lowest number of dependent equations in poselim
 	  gen besteq(0),bestvar(0); int bestpos=-1,n0deps=-1;
 	  for (int i=0;i<int(poselim.size());++i){
 	    gen curvar=elim[poselim[i]];
--- giac.orig/src/static_help.h
+++ giac/src/static_help.h
@@ -534,7 +534,7 @@
 {"blue",{"Option de la commande affichage (ou display) pour afficher en couleur.","Option of the display command to display with color.","Opción de la instrucción display para colores.","Επιλογή της εντολής display (ή affichage) για εμφάνιση χρωμάτων.",0},"Opt"," F:=display(point(2+1.5*i),red); F:=display(point(2+1.5*i),point_point+green)","display"},
 {"bohman_window",{0,"Applies the Bohman windowing function to the given signal u (or to the elements with indices between n1 and n2) and returns the result in a new list.",0,0,0},"Lst,[Interval(n1..n2)]"," scatterplot(bohman_window(randvector(1000,0..1)))","blackman_harris_window,blackman_window,bartlett_hann_window,cosine_window,gaussian_window,hamming_window,hann_poisson_window,hann_window,parzen_window,poisson_window,riemann_window,triangle_window,tukey_window,welch_window"},
 {"border",{"Renvoie la matrice formée par A bordée de b en dernière colonne, si nrows(A)=size(b), border(A,b)=tran(append(tran(A),b)).","Returns the matrix obtained by bording A with b as latest column, if nrows(A)=size(b), border(A,b)=tran(append(tran(A),b)).","Devuelve la matriz formada por A con b en el borde como última columna, si nrows(A)=size(b), border(a,b)=tran(append(tran(A),b)).","Επιστρέφει τον πίνακα που δημιουργείται από τον Α έχοντας το b ως την τελευταία στήλη, αν nrows(A)=size(b), border(A,b)=tran(append(tran(A),b)).","Gibt die Matrix zurück, die man erhält, wenn man b von rechts an A anfügt. Wenn nrows(A)=size(b), border(A,b)=tran(append(tran(A),b))."},"","border([[1,2,3,4],[4,5,6,8],[7,8,9,10]],[1,3,5]);border([[1,2,3],[4,5,6],[7,8,9]],[1,0,1])","tran,append,augment"},
-{"boxwhisker",{"Boite à moustache pour une série statistique.","Box and Whisker plot for a statistical serie.","Gráfica BoxWhisker para una serie estadística.","Σχεδιάζει το θηκόγραμμα για μια στατιστική σειρά.","Boxplot für eine statistischen Reihe. "},"","boxwhisker([-1,1,2,2.2,3,4,-2,5]);boxwhisker([1,2,3,5,10,4],x=1..2);boxwhisker([1,2,3,5,10,4],[1,2,3,1,2,3]);boxwhisker([[6,0,1,3,4,2,5],[0,1,3,4,2,5,6],[1,3,4,2,5,6,0],[3,4,2,5,6,0,1],[4,2,5,6,0,1,3],[2,5,6,0,1,3,4]])","quartiles"},
+{"boxwhisker",{"Boite à moustache pour une série statistique.","Box and Whisker plot for a statistical series.","Gráfica BoxWhisker para una serie estadística.","Σχεδιάζει το θηκόγραμμα για μια στατιστική σειρά.","Boxplot für eine statistischen Reihe. "},"","boxwhisker([-1,1,2,2.2,3,4,-2,5]);boxwhisker([1,2,3,5,10,4],x=1..2);boxwhisker([1,2,3,5,10,4],[1,2,3,1,2,3]);boxwhisker([[6,0,1,3,4,2,5],[0,1,3,4,2,5,6],[1,3,4,2,5,6,0],[3,4,2,5,6,0,1],[4,2,5,6,0,1,3],[2,5,6,0,1,3,4]])","quartiles"},
 {"break",{"Pour interrompre une boucle: while (cond) {inst1; if (cond) {inst2;break;}}.","Interrupts a loop: while (cond) {inst1; if (cond) {inst2;break;}}.","Interrumpe un bucle: while (cond) {inst1; if (cond) {inst2;break;}}.","Διακόπτει έναν βρόχο: while (cond) {inst1; if (cond) {inst2;break;}}.","Unterbricht eine Schleife: while (Bedingung) {Instruktion1; if (Bedingung) {Instruktion2;break;}}."},"","  f(a,b):={local r;while (1==1){if (b==0){break;} r:=irem(a,b);a:=b;b:=r;}a;}","continue"},
 {"breakpoint",{"Ajoute un point d'arrêt.","Adds a breakpoint.","Añade un punto de ruptura.","Προσθήκη ενός σημείου διακοπής","Fügt einen breakpoint hinzu (einen Punkt zum Stoppen)."},"","breakpoint(1)","rmbreakpoint"},
 {"brent_solver",{"Argument de fsolve indiquant la méthode pour résoudre numériquement une équation.","Argument for fsolve giving the method for solving an numerical equation.","Argumento para fsolve indicando el método para la resolución numérica de una ecuación.","Όρισμα για την εντολή fsolve που δίνει τη μέθοδο για την επίλυση μιας εξίσωσης (προσέγγιση της ρίζας).","Gibt das charakteristische Polynom von A und die Streichungsmatrizen von A-xI."},"Opt"," fsolve(cos(x)=x,x,0..1,bisection_solver); fsolve(cos(x)=x,x,0..1,brent_solver); fsolve(cos(x)=x,x,0..1,falsepos_solver); fsolve(cos(x)=x,x,0,newton_solver); fsolve(cos(x)=x,x,0,secant_solver); fsolve(cos(x)=x,x,0,steffenson_solver)","fsolve"},
@@ -546,7 +546,7 @@
 {"cSolve",{"Renvoie la liste des solutions complexes d'une équation ou la matrice dont les lignes sont solutions dans ℂ d'un système d'équations polynomiales.","Returns the list of complex solutions of an equation or a matrix where the rows are ℂ-solutions of a system of polynomial equations.","Devuelve la matriz cuyas filas son soluciones en ℂ del sistema de ecuaciones polinomiales.","Επιστρέφει τον πίνακα όπου οι γραμμές είναι λύσεις στο ℂ ενός συστήματος πολυωνυμικών εξισώσεων.",0},"LstEq,LstVar","csolve(x^4-1,x);csolve(x^4-y^4 and x+y=2,[x,y]);csolve(x^4-y^4 and x+y=0 and x^2=2*x,[x,y]);csolve(u*v-u=v and v^2=u,[u,v])","cZeros,solve,fslove"},
 {"cZeros",{"Renvoie la liste des éléments complexes qui annulent l'expression Xpr ou la matrice dont les lignes sont les solutions du système : Xpr1=0,Xpr2=0...","Returns the list of complex elements solution of Xpr=0 or the matrix where the lines are the solutions of the system : Xpr1=0,Xpr2=0...","Devuelve la lista de elementos complejos solución de Xpr=0 o la matriz donde las filas son las soluciones del sistema: Xpr1=0, Xpr2=0,..","Επιστρέφει μια λίστα από μιγαδικά στοιχεία λύσεις της Xpr=0 ή τον πίνακα οι γραμμές του οποίου είναι λύσεις του συστήματος : Xpr1=0,Xpr2=0...","Gibt die Liste der komplexen Lösungen von Xpr=0 an oder die Matrix, bei der die Zeile die Lösung des Systems: Xpr1=0, Xpr2=0,.. ist."},"","cZeros(x^2-1);cZeros([x^2-1,x^2-y^2],[x,y])","solve"},
 {"cache_tortue",{"Cache la tortue.","Hides the turtle.","Esconde la tortuga.","Αποκρύπτει τη χελώνα.","Versteckt die Schildkröte."},"","cache_tortue()","montre_tortue"},
-{"camembert",{"Trace un ou des diagramme(s) en camembert d'une série statistique à 1 variable.","Draws pie chart of a one variable statistical serie.","Muestra uno o varios gráfico(s) de torta de una serie estadística en una variable.","Σχεδιάζει το διάγραμμα πίττας μιας στατιστικής σειράς με 1 μεταβλητή.","Zeichnet ein Tortendiagramm einer statistischen Reihe mit einer Variablen."},"","camembert([[\"France\",6],[\"Allemagne\",12],[\"Suisse\",5]]);camembert([3/2,2/3,5/4,4/5,7/6,6/7,9/8,8/9,11/10]);camembert([[2,\"xyz\",\"abc\"],[\"A\",2,5],[\"B\",5,6],[\"C\",7,7]])","bar_plot"},
+{"camembert",{"Trace un ou des diagramme(s) en camembert d'une série statistique à 1 variable.","Draws pie chart of a one variable statistical series.","Muestra uno o varios gráfico(s) de torta de una serie estadística en una variable.","Σχεδιάζει το διάγραμμα πίττας μιας στατιστικής σειράς με 1 μεταβλητή.","Zeichnet ein Tortendiagramm einer statistischen Reihe mit einer Variablen."},"","camembert([[\"France\",6],[\"Allemagne\",12],[\"Suisse\",5]]);camembert([3/2,2/3,5/4,4/5,7/6,6/7,9/8,8/9,11/10]);camembert([[2,\"xyz\",\"abc\"],[\"A\",2,5],[\"B\",5,6],[\"C\",7,7]])","bar_plot"},
 {"canonical_form",{"Forme canonique du trinôme de degré 2.","Canonical_form of a 2 degree polynomial.","Forma canónica de un polinomio de grado 2.","Κανονική μορφή ενός πολυωνύμου 2ου βαθμού.","kanonische Form eines Polynoms zweiten Grades."},"","canonical_form(2*x^2-12*x+1);canonical_form(2*a^2-12*a+1,a)",""},
 {"canonical_labeling",{0,"Returns the permutation representing the canonical labeling of G.",0,0,0},"Graph(G)","canonical_labeling(graph(\"petersen\"))","isomorphic_copy,relabel_vertices"},
 {"cap",{"Retourne le cap de la tortue en degrés ou tourne la tortue selon l'argument.","Returns the turtle cap in degrees or turns the turtle with the cap given by the argument.","Devuelve la dirección de la tortuga en grados o gira la tortuga según el argumento.","Επιστρέφει την φορά (ή πορεία του καβουκιού) της χελώνας σε μοίρες ή στρέφει τη χελώνα με βάση το όρισμα που δίνεται.","Gibt die Orientierung der Schildkröte wieder in Grad oder dreht die Schildkröte abhängig vom Argument."},""," cap;cap();cap 90","position,initialise"},
@@ -781,8 +781,8 @@
 {"duration",{0,"Returns the duration of an audio clip, in seconds.",0,0,0},"Lst(clip)","duration(readwav(\"/some/file\"))","bit_depth,channels,channel_data,samplerate"},
 {"e",{"exp(1)","exp(1)","exp(1)","exp(1)","exp(1)"},""," evalf(e)","i,pi"},
 {"e2r",{"Renvoie la liste des coefficients d'un polynôme par rapport au 2ème argument ou si le 2ème argument est une liste renvoie le format interne du polynôme.","Returns the coefficients of a polynomial with respect to the 2nd argument or if the second argument is a list the internal format of the polynomial.","Devuelve los coeficientes de un polinomio con respecto al segundo argumento.","Επιστρέφει την λίστα των συντελεστών ενός πολυωνύμου αναφορικά με το 2ο όρισμα.",0},"Expr, LstVar or [Var]","symb2poly(x*3+2.1);symb2poly(3*x*y+2*y+1,y);symb2poly(3*x*y+2*y+1,x,y);symb2poly(3*x*y+2*y+1,[x,y]);symb2poly(-x^4+x*3*y+2+y^2*z,[x,y,z]);symb2poly(-x^4+x*3*y+2+y^2*z,[x,y,z])","poly2symb,r2e"},
-{"ecart_type",{"Renvoie l'écart-type d'une liste pondérée par le deuxième argument ou la liste des écarts-types des colonnes d'une matrice.","Returns the standard deviation of the elements of its argument with an optionnal second argument as pound or the list of standard deviation of the columns of a matrix.","Devuelve la desviación estándar de una lista ponderada por el segundo argumento o la lista de las desviaciones estándares de las columnas de una matriz.","Επιστρέφει την τυπική απόκλιση των στοιχείων του ορίσματος με το 2ο όρισμα ως οδηγό ή τη λίστα των τυπικών αποκλίσεων των στηλών του πίνακα.",0},"Lst||Mtrx,[Lst]","stddev([1,2,3]);stddev([1,2,3],[1,2,1]);stddev([[1,2,3],[5,6,7]])","mean,variance,stddevp"},
-{"ecart_type_population",{"Renvoie un estimateur non biaise de l'écart-type de la population dont est issu l'échantillon décrit par les éléments d'une liste pondérée par le deuxième argument.","Returns an unbiaised estimate of the population standard deviation of the sample (first argument) with an optionnal list of pounds as second argument.","Devuelve la desviación estándar de la población de la cual viene la muestra descrita por los elementos de una lista ponderada por el segundo argumento.","Επιστρέφει την πληθυσμιακή τυπική απόκλιση των στοιχείων του ορίσματος με το 2ο όρισμα ως οδηγό.",0},"Lst||Mtrx,[Lst]","stddevp([1,2,3]);stddevp([1,2,3],[1,2,1]);stddevp([[1,2,3],[5,6,7]])","mean,stddev"},
+{"ecart_type",{"Renvoie l'écart-type d'une liste pondérée par le deuxième argument ou la liste des écarts-types des colonnes d'une matrice.","Returns the standard deviation of the elements of its argument with an optional second argument as pound or the list of standard deviation of the columns of a matrix.","Devuelve la desviación estándar de una lista ponderada por el segundo argumento o la lista de las desviaciones estándares de las columnas de una matriz.","Επιστρέφει την τυπική απόκλιση των στοιχείων του ορίσματος με το 2ο όρισμα ως οδηγό ή τη λίστα των τυπικών αποκλίσεων των στηλών του πίνακα.",0},"Lst||Mtrx,[Lst]","stddev([1,2,3]);stddev([1,2,3],[1,2,1]);stddev([[1,2,3],[5,6,7]])","mean,variance,stddevp"},
+{"ecart_type_population",{"Renvoie un estimateur non biaise de l'écart-type de la population dont est issu l'échantillon décrit par les éléments d'une liste pondérée par le deuxième argument.","Returns an unbiaised estimate of the population standard deviation of the sample (first argument) with an optional list of pounds as second argument.","Devuelve la desviación estándar de la población de la cual viene la muestra descrita por los elementos de una lista ponderada por el segundo argumento.","Επιστρέφει την πληθυσμιακή τυπική απόκλιση των στοιχείων του ορίσματος με το 2ο όρισμα ως οδηγό.",0},"Lst||Mtrx,[Lst]","stddevp([1,2,3]);stddevp([1,2,3],[1,2,1]);stddevp([[1,2,3],[5,6,7]])","mean,stddev"},
 {"ecm_factor",{"Factorisation d'un entier n par la methode des courbes elliptiques, B1 parametre optionnel de friabilite","Factorization of integer n by the Elliptic Curve Method algorithm, B1 smoothness optional parameter","Calcula los vectores propios de una matriz diagonalizable.","Υπολογίζει τα ιδιοδιανύσματα (ως στήλες) ενός διαγωνιοποιήσιμου πίνακα.","Berechnet die Eigenvektoren einer diagonalisierbaren Matrix."},"Intg(a),[Intg(B1)]","ecm_factor(1000000000000000003000000000000000000000000000000000031000000000000000093)",0},
 {"ecris",{"Écris la chaîne s avec la fonte n (par défaut n=14) au point [x,y] (par défaut à la place de la tortue).","Write the string s with the fount n (by default n=14) at the point [x,y] (by default at the point where the turtle is).","Escribe la cadena s con la fuente n (por defecto n=14) en el punto [x,y] (por defecto en el lugar de la tortuga).","Γράφει την συμβολοσειρά s με μέγεθος γραμματοσειράς n (προεπιλογή n=14) στο σημείο [x,y] (προεπιλογή είναι το σημείο όπου βρίσκεται η χελώνα).","Schreibe einen String s mit der Schriftart n (ohne Spezifikation n=14) am Punkt [x,y] (ohne Spezifikation dort, wo die Schildkröte ist."},"","ecris(\"bonjour\");ecris(\"bonjour\",20);ecris(\"bonjour\",20,30,10);ecris(bonjour,20,30,10)","signe"},
 {"edge_connectivity",{0,"Returns the largest integer k for which G is k-edge connected, meaning that G remains connected after fewer than k edges are removed.",0,0,0},"graph(G)","edge_connectivity(graph(\"petersen\"));edge_connectivity(graph(\"clebsch\"));edge_connectivity(complete_graph(5))","is_two_edge_connected,is_edge_cut,vertex_connectivity"},
@@ -961,7 +961,7 @@
 {"gauche",{"Renvoie la partie gauche d'une égalité, d'un intervalle, d'une liste ou d'une chaîne.","Returns the left part of an equality, of an interval, of a list or of a string.","Devuelve la parte izquierda de una igualdad, de un intervalo, de una lista o de una cadena.","Επιστρέφει το αριστερό μέρος μια ισότητας, ενός διαστήματος, μιας λίστας ή μιας συμβολοσειράς.",0},"Equal(a=b) or Interval(a..b) or Str,Intg","left(a=b);left(x^2+1=5);left(1..5);left(\"abcdefg\",3)","right,mid,tail,head"},
 {"gauss",{"Décompose une forme quadratique en somme/différence de carrés","Split a quadratic form as a sum/difference of square","Descompone una forma cuadrática como suma/diferencia de cuadrados","Διάσπαση μιας τετραγωνικής μορφής σε άθροισμα και διαφορά τετραγώνων","Spaltet eine quadratische Form in Summen und Differenzen von Quadraten auf."},"","gauss(x^2+2*a*x*y,[x,y])","cholesky"},
 {"gauss15",{"Option de la commande area","Option of the area command","Opción del comando area","Επιλογή της εντολής area","Gibt das Polynom zurück, das durch die Liste (Nullstellen oder Pole, Multiplizität) beschrieben wird."},"Opt"," area(x^2,x=0..1,5,simpson); area(x^2,x=0..1,5,rombergt); area(x^2,x=0..1,5,rombergm);area(x^2,x=0..1,5,gauss15)","area"},
-{"gauss_seidel_linsolve",{"Résolution d'un système linéaire de matrice A et de second membre b, par la méthode itérative de Gauss-Seidel (par défaut omega=1) ou de relaxation avec une tolérance de eps et un nombre maximal d'itérations de maxiter. ","Resolution of a linear system A*X=b  by the iterative Gauss-Seidel method (by defaut omega=1) or by relaxation method, with eps as error margin and a number of iterations inferior to maxiter.",0,0,"Loesung eines linearen Gleichungssystems A*X=b mit dem Gauss-Seidel-Verfahren (ohne Spezifikition omega=1) oder dem Relaxationsverfahren, wobei die Groesse des Fehlers mit eps und die die maximale Anzahl der Iterationen mit maxiter festgelegt wird"},""," a:=[[100,2],[2,100]];gauss_seidel_linsolve(a,[0,1],1e-12);; a:=[[100,2],[2,100]];gauss_seidel_linsolve(table(a),[0,1],1e-12); ; a:=[[100,2],[2,100]];gauss_seidel_linsolve(1.5,a,[0,1],1e-12);; a:=[[100,2],[2,100]];gauss_seidel_linsolve(1.5,table(a),[0,1],1e-12); ","jacobi_linsolve,linsolve"},
+{"gauss_seidel_linsolve",{"Résolution d'un système linéaire de matrice A et de second membre b, par la méthode itérative de Gauss-Seidel (par défaut omega=1) ou de relaxation avec une tolérance de eps et un nombre maximal d'itérations de maxiter. ","Resolution of a linear system A*X=b  by the iterative Gauss-Seidel method (by default omega=1) or by relaxation method, with eps as error margin and a number of iterations inferior to maxiter.",0,0,"Loesung eines linearen Gleichungssystems A*X=b mit dem Gauss-Seidel-Verfahren (ohne Spezifikition omega=1) oder dem Relaxationsverfahren, wobei die Groesse des Fehlers mit eps und die die maximale Anzahl der Iterationen mit maxiter festgelegt wird"},""," a:=[[100,2],[2,100]];gauss_seidel_linsolve(a,[0,1],1e-12);; a:=[[100,2],[2,100]];gauss_seidel_linsolve(table(a),[0,1],1e-12); ; a:=[[100,2],[2,100]];gauss_seidel_linsolve(1.5,a,[0,1],1e-12);; a:=[[100,2],[2,100]];gauss_seidel_linsolve(1.5,table(a),[0,1],1e-12); ","jacobi_linsolve,linsolve"},
 {"gaussian_window",{0,"Applies the Gaussian windowing function with parameter 0<a<=0.5 (by default a=0.1) to the given signal u (or to the elements with indices between n1 and n2) and returns the result in a new list.",0,0,0},"Lst,[Real(a)],[Interval(n1..n2)]"," scatterplot(gaussian_window(randvector(1000,0..1),0.4))","blackman_harris_window,blackman_window,bohman_window,cosine_window,bartlett_hann_window,hamming_window,hann_poisson_window,hann_window,parzen_window,poisson_window,riemann_window,triangle_window,tukey_window,welch_window"},
 {"gaussjord",{"Réduction sous forme échelonnée de AX=b (M=A|(-b)) [Réduction sur les k premières colonnes].","Row reduction to echelon form of AX=b (M=A|(-b)) [Reduction on columns 0..k-1]","Reducción a la forma escalonada de AX=b (M=A|(-b)) [Reducción sobre los columnas 0..k-1].","Αναγωγή σε κλιμακωτή μορφή του AX=b (M=A|(-b)).",0},"Mtrx(M),[Intg(k)]||Opt","rref([[3,1,-2],[3,2,2]]);rref([[2,1,1,-1],[1,1,2,-1],[1,2,1,-4]]);rref([[2,1,1,-1],[1,1,2,-1],[1,2,1,-4]],2);rref([[1,1,0,0,-a1],[0,1,1,0,-a2],[0,0,1,1,-a3],[1,0,0,1,-a4]],keep_pivot)","ker,image,det,Rref,pivot,ref,keep_pivot"},
 {"gaussquad",{"Renvoie la valeur approchée de integrate(f(x),x,a,b) par une quadrature de Gauss adaptative à 15 points.","Returns the approximate value of integrate(f(x),x,a,b) by adaptive Gaussian quadratures with 15 points.","Devuelve el valor aproximado de integrate(f(x),x,a,b)  por el método de Gauss.","Επιστρέφει την προσεγγιστική τιμή του ολοκληρώματος(f(x),x,a,b) με τη μέθοδο του Gauss.","Gibt den ungefähren Wert von integrate(f(x),x,a,b) durch adaptive Gauß-Quadratur mit 15 Punkten zurück."},"","gaussquad(exp(x^2),x,0,1);gaussquad(x^2,x,0,1);gaussquad(exp(-x^2),x,-1,1)","integrate,romberg"},
@@ -1065,7 +1065,7 @@
 {"highlight_vertex",{0,"Change the color of vertex v resp. colors of vertices from V in G to c resp. C (green by default) and return the modified copy of G.",0,0,0},"Graph(G),Vrtx(v)||Lst(V),[Color(c)||Lst(C)]"," draw_graph(highlight_vertex(cycle_graph(3),1))","highlight_edges,highlight_subgraph,highlight_trail"},
 {"highpass",{0,"Returns the result of applying a simple first-order highpass RC filter with cutoff frequency c (the default samplerate is 44100) to the given signal s.",0,0,0},"Lst(s),Real(c),[Intg(samplerate)]"," f:=unapply(periodic(sign(x),x,-1/880,1/880),x):;s:=createwav(apply(f,soundsec(1))):;playsnd(highpass(s,5000))","lowpass,moving_average"},
 {"hilbert",{"Renvoie la n-ième matrice de Hilbert : Hjk=1/(j+k+1) j,k=1..n","Returns the order n Hilbert matrix : Hjk=1/(j+k+1) j,k=1..n","Devuelve la matriz n-ésima de Hilbert: Hjk=1/(j+k+1) j,k=1..n","Επιστρέφει τον n-στής τάξης πίνακα Hilbert : Hjk=1/(j+k+1) j,k=1..n","Gibt die Hilbert-Matrix der Ordnung n zurück: Hjk=1/(j+k+1) j,k=1..n"},"","hilbert(4)",""},
-{"histogram",{"Trace l'histogramme des données de data, on peut préciser une liste d'effectifs, ou un nombre nc de classes ou le mimimum classmin des classes et la largeur classsize des classes.","Draws the histogram of data, optional arguments are eff (number of data for each data element) or nc the number of classes or the classes minimum and size","Traza el histograma del argumento.","Σχεδιάζει το ιστόγραμμα του ορίσματος","Zeichnet das Histogramm von Daten. Optionale Argumente sind eff (Zahl der Daten für jedes Datenelement) oder nc (Anzahl der Klassen) oder classmin (Minimum der Klassen) oder classsize (Größe der Klasse)."},"","histogram([1,2,1,1,2,1,2,4,3,3]);histogram([1,2,1,1,2,1,2,4,3,3],0.5,1);histogram(seq(rand(1000),k,0,100),0,100);histogram(binomial,10,.5);histogram([[0,1],[1,4],[2,3],[3,2],[4,1]]);histogram([[1.5..1.65,50],[1.65..1.7,20],[1.7..1.8,30]]);histogram(seq(rand(1000),k,0,100),0,100);histogram(seq(rand(1000),k,0,100),10)","cumulated_frequencies,classes,bar_plot,frequencies"},
+{"histogram",{"Trace l'histogramme des données de data, on peut préciser une liste d'effectifs, ou un nombre nc de classes ou le minimum classmin des classes et la largeur classsize des classes.","Draws the histogram of data, optional arguments are eff (number of data for each data element) or nc the number of classes or the classes minimum and size","Traza el histograma del argumento.","Σχεδιάζει το ιστόγραμμα του ορίσματος","Zeichnet das Histogramm von Daten. Optionale Argumente sind eff (Zahl der Daten für jedes Datenelement) oder nc (Anzahl der Klassen) oder classmin (Minimum der Klassen) oder classsize (Größe der Klasse)."},"","histogram([1,2,1,1,2,1,2,4,3,3]);histogram([1,2,1,1,2,1,2,4,3,3],0.5,1);histogram(seq(rand(1000),k,0,100),0,100);histogram(binomial,10,.5);histogram([[0,1],[1,4],[2,3],[3,2],[4,1]]);histogram([[1.5..1.65,50],[1.65..1.7,20],[1.7..1.8,30]]);histogram(seq(rand(1000),k,0,100),0,100);histogram(seq(rand(1000),k,0,100),10)","cumulated_frequencies,classes,bar_plot,frequencies"},
 {"hold",{"Retourne son argument sans l'évaluer (et aussi a:=quote(a) qui purge a).","Returns its argument unevaluated (and also a:=quote(a) that purge a).","Devuelve su argumento sin evaluarlo (y también a:=quote(a) que purge a).","Αναστολή υπολογισμών. Επιστρέφει το όρισμα μη-αποτιμημένο (και επίσης a:=quote(a) καθαρίζει το a).","Boolesche Auswertung des Arguments."},"Expr","quote(1+2);quote(1/x+1/(x-1));quote((x+1)*(x-1))",""},
 {"homothety",{"homothetie(C,k,A)=point A1 tel que vect(C,A1)=k*vect(C,A) i.e en 2d est la similitude de centre C, de rapport abs(k) et d'angle arg(k).","homothety(C,k,A)=point A1 such as vect(C,A1)=k*vect(C,A) i.e in 2d it is the similarity center C, coeff abs(k) and angle arg(k).","homothety(C,k,A)=punto A1 tal que vect(C,A1)=k*vect(C,A) es decir, en 2o es la semejanza de centro C, de coeficiente abs(k) y de ángulo arg(k).","homothety(C,k,A)=σημείο Α1 τέτοιο ώστε vect(C,A1)=k*vect(C,A) π.χ. σε 2-Δ είναι το κέντρο ομοιότητας C, αναφορικά με abs(k) και γωνίας arg(k).","homothety(C,k,A)=Punkt A1 sodass vect(C,A1)=k*vect(C,A) in 2D ist es also das Ähnlichkeitszentrum C, abs(k) ist das Verhältnis und arg(k) ist der Winkel."},"","homothety(1+i,1/3,i);homothety(point(1,1,1),1/3,point(0,1,0)); h:=homothety(1+i,1/3);h(i); h:=homothety(point(1,1,1),1/3);h(point(0,1,0))","similarity,inversion"},
 {"horner",{"Renvoie la valeur P(a) calculée selon la méthode de Hörner. La syntaxe horner(liste_alpha_i,liste_x_i,x) permet d'utiliser les differences divisees pour evaluer un polynome d'interpolation en x.","Returns the value of P(a) calculated with Horner's method. With horner(liste_alpha_i,liste_x_i,x), evals an interpolation polynomial from the divided differences at x.","Devuelve el valor de P(a) calculado según el método de Horner.","Επιστρέφει την τιμή P(a) υπολογισμένη με τη μέθοδο Horner.","Gibt den Wert von P(a) berechnet durch das Horner-Schema zurück. horner(liste_alpha_i,liste_x_i,x) erlaubt es die Auswertung mit Hilfe der geteilten Differenzen durchzuführen, die ein Interpolationspolynom bei x berechnen. "},"","horner(x^2+1,2);horner([1,0,1],2);horner(x^2+y*x+y^3-1,2,y); X:=[0.0,1.0,2.0]; A:=lagrange(X,exp,lagrange); horner(A,X,1.5);","convert,base,revlist"},
@@ -1286,7 +1286,7 @@
 {"lineariser_trigo",{"Linéarisation trigonométrique","Trigonometric linearization","Linealización trigonométrica","Τριγωνομετρική γραμμικοποίηση","Gibt die Reihe n oder die Folge der Reihen n1..n2 der Matrix A zurück, oder mit den optionalen Argumenten: count,count_eq,count_inf,count_sup."},"ExprTrig","tlin(sin(x)^3);tlin(cos(x)*cos(y))","texpand,lin"},
 {"linfnorm",{"Triple norme d'une matrice subordonnée à la norme linfini.","Matrix norm induced by linfinity norm.","Devuelve el valor de epsilon de la configuración del CAS.","Επιστρέφει την τιμή του έψιλον της ρύθμισης του CAS.","Gibt den Wert von Epsilon der CAS-Konfiguration zurück."},"Mtrx","linfnorm([[1,2,3],[3,-9,6],[4,5,6]])","l1norm,l2norm,matrix_norm,frobenius_norm"},
 {"linsolve",{"Résolution d'un système d'équations linéaires.","Linear equations system solver.","Resolución de un sistema de ecuaciones lineales.","Επιλύει σύστημα γραμμικών εξισώσεων.","Löst lineare Gleichungssysteme."},"","linsolve([x+y+z=1,x-y=2,2*x-z=3],[x,y,z]);linsolve([m*x+y=a,x+m*y=b],[x,y]);linsolve([x+y-z-1,x-y+1,x-y-z-1]%2,[x,y,z]);linsolve([[3,4],[1,2]],[0,1]); p,l,u:=lu([[3,4],[1,2]]); linsolve(p,l,u,[0,1]) ;linsolve([2*x+y+z=1,x+y+2*z=1,x+2*y+z=4],[x,y,z]);linsolve([[2,1,1],[1,1,2],[1,2,1]],[1,1,4]); p,l,u:=lu([[2,1,1],[1,1,2],[1,2,1]]);linsolve(p,l,u,[1,1,4]); a:=[[100,2],[2,100]];linsolve(evalf(a),[0,1]);","solve,proot,simult,gaussjord,pivot,ref,conjugate_gradient"},
-{"linspace",{"linspace(a,b,n) génère une liste de n éléments régulièrement espacés entre a et b, par défaut n=100 (commande compatible scilab).","linspace(a,b,n) creates a list of n elements at regular intervals between a and b, by defaut n=100 (compatibility with scilab).",0,0,"linspace(a,b,n) generiert eine Liste aus n aequidistanten Elementen zwischen a und b, ohne Spezifikation n=100 (kompatibel mit scilab)"},"","linspace(0,1,11)",0},
+{"linspace",{"linspace(a,b,n) génère une liste de n éléments régulièrement espacés entre a et b, par défaut n=100 (commande compatible scilab).","linspace(a,b,n) creates a list of n elements at regular intervals between a and b, by default n=100 (compatibility with scilab).",0,0,"linspace(a,b,n) generiert eine Liste aus n aequidistanten Elementen zwischen a und b, ohne Spezifikation n=100 (kompatibel mit scilab)"},"","linspace(0,1,11)",0},
 {"lis",{"Instruction dans un programme permettant l'entrée d'une expression qui sera stockée dans var.","Instruction in a program so that you can input an expression and store it in var.","Instrucción en un programa que permite introducir una expresión y guardarla en var.","Εντολή σε ένα πρόγραμμα για την εισαγωγή μιας παράστασης και αποθήκευση στη var.",0},"[Str],Var(Var)","input(\"age\",a)","InputStr,output"},
 {"lis_phrase",{"Instruction dans un programme permettant l'entrée d'une chaîne de caractères qui sera stockée dans var.","Instruction in a program so that you can input a string and store it in var.","Instrucción en un programa que permite introducir una cadena de caracteres y guardarla en var.","Εντολή σε ένα πρόγραμμα για την εισαγωγή μιας συμβολοσειράς που θα αποθηκευτεί στη var.","(S,A,L,O):=lll(M), Kurzbasis eines Gitters (L*O=A*M=S)."},"[Str],Var(Var)","InputStr(\"nom?\",N)","input"},
 {"list",{"Conversion en liste.","Convert to a list","Opción del comando convert o convertir (n-poly=>list).","Επιλογή της εντολής convert ή convertir (n-poly=>list).","Option des convert oder convertir Befehls (n-poly=>list)."},"","list((1,2))","symb2poly,convert"},
@@ -1402,7 +1402,7 @@
 {"mods",{"Retourne le reste symétrique de la division euclidienne de 2 entiers.","Returns the Euclidean symmetric remainder of two integers.","Devuelve el resto simétrico de la división euclidiana de 2 enteros.","Επιστρέφει το Ευκλείδιο συμμετρικό υπόλοιπο 2 ακεραίων.",0},"Intg,Intg","smod(8,3);smod(10,4);smod(11,7)","irem,iquo,mod,fracmod"},
 {"monotonic",{"Renvoie un reel qui croit strictement en fonction du temps ecoule","Returns a real that increases as time passes","En un programa, crea entradas: Dialog{Title(\"título\");Text(\"Alumno\"+string(c));Request(\"Nombre=\",a);Request(\"Edad=\",b);DropDown(\"Nivel\",[\"un\",\"dos\"],n);} (se puede reemplazar {} con EndDlog al final).","Σε ένα πρόγραμμα, εκτελεί καταχωρήσεις :Dialog{Title(\"title\");Text(\"Pupil\"+ string(c));Request(\"Name=\",a);Request(\"Age=\",b);DropDown(\"Level\",[\"un\",\"deux\"],n);} (το {} μπορεί να αντικατασταθεί με EndDlog στο τέλος).","Schneidet das Polynom P bei Ordnung n ab."},"","monotonic()",0},
 {"montre_tortue",{"Montre la tortue.","Shows the turtle.","Muestra la tortuga.","Εμφανίζει τη χελώνα.","Zeigt die Schildkröte."},"","montre_tortue()","cache_tortue"},
-{"moustache",{"Boite à moustache pour une série statistique.","Box and Whisker plot for a statistical serie.","Gráfica BoxWhisker para una serie estadística.","Σχεδιάζει το θηκόγραμμα για μια στατιστική σειρά.","A=B oder equal(A,B) wird in die Liste [A,B] umgewandelt."},"Lst,[Lst],[x=a..b||y=a..b]","boxwhisker([-1,1,2,2.2,3,4,-2,5]);boxwhisker([1,2,3,5,10,4],x=1..2);boxwhisker([1,2,3,5,10,4],[1,2,3,1,2,3]);boxwhisker([[6,0,1,3,4,2,5],[0,1,3,4,2,5,6],[1,3,4,2,5,6,0],[3,4,2,5,6,0,1],[4,2,5,6,0,1,3],[2,5,6,0,1,3,4]])","quartiles"},
+{"moustache",{"Boite à moustache pour une série statistique.","Box and Whisker plot for a statistical series.","Gráfica BoxWhisker para una serie estadística.","Σχεδιάζει το θηκόγραμμα για μια στατιστική σειρά.","A=B oder equal(A,B) wird in die Liste [A,B] umgewandelt."},"Lst,[Lst],[x=a..b||y=a..b]","boxwhisker([-1,1,2,2.2,3,4,-2,5]);boxwhisker([1,2,3,5,10,4],x=1..2);boxwhisker([1,2,3,5,10,4],[1,2,3,1,2,3]);boxwhisker([[6,0,1,3,4,2,5],[0,1,3,4,2,5,6],[1,3,4,2,5,6,0],[3,4,2,5,6,0,1],[4,2,5,6,0,1,3],[2,5,6,0,1,3,4]])","quartiles"},
 {"moving_average",{0,"Applies a moving average filter of length n to a signal sample A, and returns its result as an array of length nops(A)-n+1.",0,0,0},"Lst(A),Intg(n)"," snd:=soundsec(2):;data:=0.5*threshold(3*sin(2*pi*220*snd),[-1.0,1.0])+randvector(length(snd),normald,0,0.05):;moving_average(data,25)","lowpass"},
 {"moyal",{"Produit de Moyal de 2 symboles.","Moyal product of 2 symbols.","Producto de Moyal de 2 expresiones simbólicas.","Γινόμενο Moyal 2 συμβόλων.","Moyal-Produkt zweier Symbole."},"","moyal(x^2+y^4,x^4-y^2,[x,y],5)",""},
 {"moyenne",{"Moyenne d'une liste pondérée par le deuxième argument ou des colonnes d'une matrice.","Mean of a list with the second argument as pound or of the columns of a matrix.","Promedio de una lista ponderada, con el segundo argumento como peso, o de las columnas de una matriz.","Μέση τιμή μιας λίστας με το δεύτερο όρισμα ως οδηγό ή των στηλών ενός πίνακα.",0},"Lst||Mtrx,[Lst]","mean([1,2,3]);mean([1,2,3],[1,2,3]);mean([[1,2,3],[1,2,3]])","stddev"},
@@ -1654,7 +1654,7 @@
 {"randNorm",{"Renvoie un réel aléatoirement distribué selon la loi normale N(mu,sigma).","Returns a random real with normal distribution N(mu,sigma).","Devuelve un número aleatorio distribuido según la ley normal N(mu,sigma).","Επιστρέφει έναν τυχαίο πραγματικό με κανονική κατανομή N(μ,σ).","Gibt das kleinste gemeinsame Vielfache von 2 Polynomen mit mehreren Variablen oder 2 ganzen Zahlen oder 2 rationalen Zahlen zurück. "},"Real(mu),Real(sigma)","randnorm(0,1);randnorm(2,1)","rand,randpoly,ranm,randvector,randexp,randbinomial,randpoisson,randgeometric,randmultinomial"},
 {"randPoly",{"Renvoie un polynôme de variable var (ou x), de degré n et dont les coefficients sont des entiers aléatoires équirépartis sur -99,+99 ou selon une loi.","Returns a polynomial of variable var (or x), of degree n and where the coefficients are random integers in the range -99 through 99 with uniform distribution or according to a law.","Devuelve el polinomio de variable var (o x), de grado n y cuyos coeficientes son enteros aleatorios equirepartidos entre -99 y 99.","Επιστρέφει ένα πολυώνυμο μεταβλητής var (ή x), βαθμού n και του οποίου οι συντελεστές είναι τυχαίοι ακέραιοι από -99 μέχρι 99 ομοιόμορφα κατανεμημένοι.","Gibt den Rest der Division eines Polynoms durch eine Gröbnerbasis zurück."},"[Var(Var)],Intg(n),[law]","randpoly(5);randpoly(t,8);randpoly(t,8,-1..1);randpoly([x,y],[10,3]);randpoly([x,y],[10,3],1 mod 7); GF(2,8,g);randpoly(t,8,g);randpoly([x,y],[2,3],g)","ranm,randvector"},
 {"randbetad",{"Renvoie un réel aléatoirement distribué selon la loi Beta de paramètre a>0 et b>0","Returns a random real according to the Beta distribution of parameter a>0 and b>0","Devuelve un número aleatorio distribuido según la ley Beta (parámetro a>0, b>0).","Επιλογή της εντολής display (ή affichage) για μια γραμμή.",0},"Real(a),Real(b)","randbetad(1,2);randbetad(1.5,4)","rand,randpoly,randnorm,randvector"},
-{"randbinomial",{"Renvoie un entier aléatoirement distribué selon la loi binomiale B(n,p) i.e. le nombre de succès dans une suite de n tirages indépendants lorsque pour chaque tirage, un succès est de probabilité p.","Returns a random integer with binomial distribution B(n,p) i.e. the number of success for n independant tests where for each test, the success probability is p.","Devuelve un número aleatorio distribuido según la ley binomimal B(n,p).","Επιστρέφει έναν τυχαίο πραγματικό με κατανομή B(n,p).","Gibt ein zufällige ganze Zahl zurück, der eine binomial-Verteilung B(n,p) zugrunde liegt d.h. die Anzahl der Erfolge für n unabhängige Tests, wobei jeder Test die Wahrscheinlichkeit p hat positiv auszufallen. "},"","randbinomial(10,0.4);randbinomial(100,0.8)","rand,randpoly,ranm,randvector,randexp,randnorm,randpoisson,randgeometric,randmultinomial"},
+{"randbinomial",{"Renvoie un entier aléatoirement distribué selon la loi binomiale B(n,p) i.e. le nombre de succès dans une suite de n tirages indépendants lorsque pour chaque tirage, un succès est de probabilité p.","Returns a random integer with binomial distribution B(n,p) i.e. the number of success for n independent tests where for each test, the success probability is p.","Devuelve un número aleatorio distribuido según la ley binomimal B(n,p).","Επιστρέφει έναν τυχαίο πραγματικό με κατανομή B(n,p).","Gibt ein zufällige ganze Zahl zurück, der eine binomial-Verteilung B(n,p) zugrunde liegt d.h. die Anzahl der Erfolge für n unabhängige Tests, wobei jeder Test die Wahrscheinlichkeit p hat positiv auszufallen. "},"","randbinomial(10,0.4);randbinomial(100,0.8)","rand,randpoly,ranm,randvector,randexp,randnorm,randpoisson,randgeometric,randmultinomial"},
 {"randchisquare",{"Renvoie un entier aléatoirement distribué selon la loi du chi^2 ayant n degrés de liberté, χ^2(n).","Returns a random integer with chi^2 distribution, χ^2(n).","Devuelve un número aleatorio distribuido según la ley Chi-cuadrado, χ^2(n).","Επιστρέφει έναν τυχαίο πραγματικό με κατανομή χ^2(n).","Gibt eine zufällige ganze Zahl zurück, der die chi^2-Verteilung chi^2(n) zugrunde liegt (n Anzahl der Freiheitsgrade)."},"","randchisquare(5);randchisquare(2)","rand,randpoly,ranm,randvector,randexp,randbinomial,randnorm,randgeometric,randmultinomial"},
 {"randexp",{"Renvoie un réel aléatoirement distribué selon la loi exponentielle de paramètre a>0.","Returns a random real according to the exponential distribution of parameter a>0.","Devuelve un número aleatorio distribuido según la ley exponencial (parámetro a>0).","Επιστρέφει έναν τυχαίο πραγματικό με Εκθετικό κατανομή (παράμετρο a>0).","Gibt die zufällige reelle Zahl mit Exponential-Verteilung mit Parameter a>0 zurück."},"","randexp(1);randexp(2)","rand,randpoly,randnorm,randvector"},
 {"randfisher",{"Renvoie un entier aléatoirement distribué selon la loi de Fisher-Snedecor F(n,m).","Returns a random integer with Fisher-Snedecor distribution F(n,m).","Devuelve un número aleatorio distribuido según la ley Fisher-Snedecor F(n,m).","Επιστρέφει έναν τυχαίο πραγματικό με κατανομή Fisher-Snedecor F(n,m).","Gibt die zufällige ganze Zahl mit Fisher-Snedecor-Verteilung F(n,m) zurück."},"","randfisher(5,2);randfisher(2,4)","rand,randpoly,ranm,randvector,randexp,randbinomial,randnorm,randgeometric,randmultinomial"},
@@ -1875,9 +1875,9 @@
 {"star_graph",{0,"Returns the complete bipartite graph complete_graph(1,n).",0,0,0},"Intg(n)","star_graph(5)","complete_graph,wheel_graph"},
 {"star_point",{"Option de la commande affichage (ou display) pour un point.","Option of the display command for a point.","Opción de la instrucción display para un punto.","Επιλογή της εντολής display (ή affichage) για ένα σημείο.","equation gibt die kartesische Gleichung einer Kurve an. "},"Opt"," F:=display(point(2+1.5*i),point_point); F:=display(point(2+1.5*i),rhombus_point)","display"},
 {"start",{"Boucle en mode RPN.","RPN loop.","Bucle en modo RPN.","Βρόχος σε λειτουργία RPN (Αντίστροφος Πολωνικός Συμβολισμός).","RPN Schleife."},"",0,""},
-{"stdDev",{"Renvoie un estimateur non biaise de l'écart-type de la population dont est issu l'échantillon décrit par les éléments d'une liste pondérée par le deuxième argument.","Returns an unbiaised estimate of the population standard deviation of the sample (first argument) with an optionnal list of pounds as second argument.","Devuelve la desviación estándar de la población de la cual viene la muestra descrita por los elementos de una lista ponderada por el segundo argumento.","Επιστρέφει την πληθυσμιακή τυπική απόκλιση των στοιχείων του ορίσματος με το 2ο όρισμα ως οδηγό.",0},"Lst||Mtrx,[Lst]","stddevp([1,2,3]);stddevp([1,2,3],[1,2,1]);stddevp([[1,2,3],[5,6,7]])","mean,stddev"},
-{"stddev",{"Renvoie l'écart-type d'une liste pondérée par le deuxième argument ou la liste des écarts-types des colonnes d'une matrice.","Returns the standard deviation of the elements of its argument with an optionnal second argument as pound or the list of standard deviation of the columns of a matrix.","Devuelve la desviación estándar de una lista ponderada por el segundo argumento o la lista de las desviaciones estándares de las columnas de una matriz.","Επιστρέφει την τυπική απόκλιση των στοιχείων του ορίσματος με το 2ο όρισμα ως οδηγό ή τη λίστα των τυπικών αποκλίσεων των στηλών του πίνακα.","Gibt die Standardabweichung der Elemente im Argument zurück, die mit dem optionalen zweiten Argument gewichtet werden können. Sonst kann auch die Liste der Standardabweichungen der Spalten einer Matrix berechnet werden. "},"","stddev([1,2,3]);stddev([1,2,3],[1,2,1]);stddev([[1,2,3],[5,6,7]])","mean,variance,stddevp"},
-{"stddevp",{"Renvoie un estimateur non biaise de l'écart-type de la population dont est issu l'échantillon décrit par les éléments d'une liste pondérée par le deuxième argument.","Returns an unbiaised estimate of the population standard deviation of the sample (first argument) with an optionnal list of pounds as second argument.","Devuelve la desviación estándar de la población de la cual viene la muestra descrita por los elementos de una lista ponderada por el segundo argumento.","Επιστρέφει την πληθυσμιακή τυπική απόκλιση των στοιχείων του ορίσματος με το 2ο όρισμα ως οδηγό.","Gibt eine erwartungstreue Schätzung der Standardabweichung der Population der Stichprobe (erstes Argument). Im optionalen zweiten Argument kann eine Liste von Gewichtungen angegeben werden. "},"","stddevp([1,2,3]);stddevp([1,2,3],[1,2,1]);stddevp([[1,2,3],[5,6,7]])","mean,stddev"},
+{"stdDev",{"Renvoie un estimateur non biaise de l'écart-type de la population dont est issu l'échantillon décrit par les éléments d'une liste pondérée par le deuxième argument.","Returns an unbiaised estimate of the population standard deviation of the sample (first argument) with an optional list of pounds as second argument.","Devuelve la desviación estándar de la población de la cual viene la muestra descrita por los elementos de una lista ponderada por el segundo argumento.","Επιστρέφει την πληθυσμιακή τυπική απόκλιση των στοιχείων του ορίσματος με το 2ο όρισμα ως οδηγό.",0},"Lst||Mtrx,[Lst]","stddevp([1,2,3]);stddevp([1,2,3],[1,2,1]);stddevp([[1,2,3],[5,6,7]])","mean,stddev"},
+{"stddev",{"Renvoie l'écart-type d'une liste pondérée par le deuxième argument ou la liste des écarts-types des colonnes d'une matrice.","Returns the standard deviation of the elements of its argument with an optional second argument as pound or the list of standard deviation of the columns of a matrix.","Devuelve la desviación estándar de una lista ponderada por el segundo argumento o la lista de las desviaciones estándares de las columnas de una matriz.","Επιστρέφει την τυπική απόκλιση των στοιχείων του ορίσματος με το 2ο όρισμα ως οδηγό ή τη λίστα των τυπικών αποκλίσεων των στηλών του πίνακα.","Gibt die Standardabweichung der Elemente im Argument zurück, die mit dem optionalen zweiten Argument gewichtet werden können. Sonst kann auch die Liste der Standardabweichungen der Spalten einer Matrix berechnet werden. "},"","stddev([1,2,3]);stddev([1,2,3],[1,2,1]);stddev([[1,2,3],[5,6,7]])","mean,variance,stddevp"},
+{"stddevp",{"Renvoie un estimateur non biaise de l'écart-type de la population dont est issu l'échantillon décrit par les éléments d'une liste pondérée par le deuxième argument.","Returns an unbiaised estimate of the population standard deviation of the sample (first argument) with an optional list of pounds as second argument.","Devuelve la desviación estándar de la población de la cual viene la muestra descrita por los elementos de una lista ponderada por el segundo argumento.","Επιστρέφει την πληθυσμιακή τυπική απόκλιση των στοιχείων του ορίσματος με το 2ο όρισμα ως οδηγό.","Gibt eine erwartungstreue Schätzung der Standardabweichung der Population der Stichprobe (erstes Argument). Im optionalen zweiten Argument kann eine Liste von Gewichtungen angegeben werden. "},"","stddevp([1,2,3]);stddevp([1,2,3],[1,2,1]);stddevp([[1,2,3],[5,6,7]])","mean,stddev"},
 {"steffenson_solver",{"Argument de fsolve indiquant la méthode pour résoudre numériquement une équation.","Argument for fsolve giving the method for solving an numerical equation.","Argumento para fsolve indicando el método para la resolución numérica de una ecuación.","Όρισμα για την εντολή fsolve που δίνει τη μέθοδο για την επίλυση μιας εξίσωσης (προσέγγιση της ρίζας).","Liste der Variablen eines Objektes (mit rationaler Abhängigkeit). "},"Opt"," fsolve(cos(x)=x,x,0..1,bisection_solver); fsolve(cos(x)=x,x,0..1,brent_solver); fsolve(cos(x)=x,x,0..1,falsepos_solver); fsolve(cos(x)=x,x,0,newton_solver); fsolve(cos(x)=x,x,0,secant_solver); fsolve(cos(x)=x,x,0,steffenson_solver)","fsolve"},
 {"step",{"Saut de la variable dans une itération (taille du saut).","Step in an iteration (step size of incrementation).","Incremento en una iteración (tamaño del salto).","Μέγεθος βήματος μεταβλητής μέσα σε βρόχο.","Schrittgröße in einer Iteration"},""," s:=0;for j from 1 to 10 step 2 do s:=s+j end_for; s:=0;for j from 1 step 2 to 10 do s:=s+j end_for; s:=0;for j from 1 to 10 by 2 do s:=s+j end_for; s:=0;for j from 1 by 2 to 10 do s:=s+j end_for; s:=0;pour j de 1 jusque 10 pas 2 do s:=s+j fpour; s:=0;pour j de 1 pas 2 jusque 10 do s:=s+j fpour","for,pour"},
 {"stereo2mono",{0,"Returns an audio clip with all channels in the input clip downmixed to a single one.",0,0,0},"Lst(clip)","stereo2mono(readwav(\"/some/file\"))","channel_data,mean,createwav"},
